src/main/frontend/fs/sync.cljs

```diff
@@ -1539,6 +1561,63 @@
                            delete-filetxns)]
     (set (concat update-file-items rename-file-items delete-file-items))))
 
+
+(defn- <fetch-remote-and-update-local-files
+  [graph-uuid base-path relative-paths]
+  (go
+    (let [fetched-file-rpaths (<! (<fetch-remote-files rsapi graph-uuid base-path relative-paths))]
+      (p->c (p/all (->> fetched-file-rpaths
+                        (map (fn [rpath]
+                               (p/let [incoming-file (path/path-join "logseq/version-files/incoming" rpath)
```

Comment:
path/path-join usage like this(first arg: `xxx/yyy/zzz`) is all platform-compatible?

Reply:
Yes. But it cannot handle UNC paths on windows.